SampleApp/SearchPinsViewController.swift

```diff
@@ -9,16 +9,23 @@
 import Pelias
 import TangramMap
 
-class SearchPinsViewController: SampleMapViewController, UITextFieldDelegate {
+class SearchPinsViewController: SampleMapViewController, UITextFieldDelegate, AutocompleteSearchDelegate {
 
   @IBOutlet weak var searchField: UITextField!
 
+  @IBOutlet weak var displaySearch: UIButton!
+  let searchListSegueId = "searchListSegueId"
+
   override func viewDidLoad() {
     super.viewDidLoad()
 
-    try? loadStyle(.bubbleWrap)
+    try? loadStyleAsync(.bubbleWrap) { [unowned self] (style) in
```

Comment:
Since the render loop is, in fact, running during an async load, we're probably rendering an "empty" scene, with the default background color of `(0, 0, 0)`. There are things we could do about this.